language.md

```diff
@@ -131,13 +131,14 @@ Na kama lugha yako ni Swahili, [click hapa](swahili/swahili.md)
 
 As jy Afrikaans kan praat, gaan [hier](afrikaans/kos.md)
 
-=======
-Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
-
-If you speak Gujarati, go [here](Gujarati/gujarati.md).
-
-If you want learn Telugu, click [here](Telugu/Telugu.md).
-
-If you want to know more about IoT, click [here](english/iot/intro.md).
-
-If GibberisHkjsbckjsjcn,[hereeekdcskj](Gibberish/Gibberish.md)
+Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
+
+If you speak Gujarati, go [here](Gujarati/gujarati.md).
+
+If you want learn Telugu, click [here](Telugu/Telugu.md).

```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.